cmd/lncli/cmd_intercept.go

```diff
@@ -0,0 +1,84 @@
+package main
```

Comment:
Please don't forget to remove this commit.

Reply:
Good one! Removed.